tests/Makefile.am

```diff
@@ -1,4 +1,5 @@
 DIST_SUBDIRS = ftests gunit
 if WITH_TESTS
-SUBDIRS = $(DIST_SUBDIRS)
+SUBDIRS = gunit
+#SUBDIRS = $(DIST_SUBDIRS)
```

Comment:
This feels like an oops :)

Reply:
My bad, was testing some bug with Gunit.